src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -741,13 +741,22 @@
                 and (../gmd:accessConstraints/gmd:MD_RestrictionCode/@codeListValue = 'RI_609'
                 or ../gmd:useConstraints/gmd:MD_RestrictionCode/@codeListValue = 'RI_609')) or
 
+                (../gmd:accessConstraints/gmd:MD_RestrictionCode/@codeListValue != 'RI_609'
```

Comment:
I don't understand the extra or which seem to indicate that if the other constraints is not specified in 'Access Constraints' or 'Use Constraints' then it is ok to add other constraints.    I don't believe this correct...

I believe the user can only add other constraints if they specify other constraints in  'Access Constraints' or 'Use Constraints'